src/config/keybind.c

```diff
@@ -124,6 +124,25 @@ keybind_create(const char *keybind)
 	gchar **symnames = g_strsplit(keybind, "-", -1);
 	for (size_t i = 0; symnames[i]; i++) {
 		char *symname = symnames[i];
+		/*
+		 * Since "-" is used as a separator, a keybind string like "W--"
+		 * becomes "W", "", "". This means that it is impossible to bind
+		 * an action to the "-" key in any way.
```

Comment:
This is not technically correct, it can be defined like `W-minus` or `W-0x21` (in for example the `us` layout) so I suggest `s/any way/this way/`.

Reply:
Done.